data/nav.yml

```diff
@@ -391,6 +391,8 @@
         - name: "Cluster queue metrics"
           path: "pipelines/cluster-queue-metrics"
           pill: "beta"
+    - name: "Service Quotas"
+      path: "pipelines/service-quotas"
```

Comment:
I note you've got this under Pipelines, but a handful of the limits are not pipelines-specific. I wonder if we need a global page? or just limit what's on this one to pipelines ones?

Reply:
Perhaps we do need a global page. At the moment, our navigation has:
- [Pipelines](https://buildkite.com/docs/pipelines)
- [Test Analytics](https://buildkite.com/docs/test-analytics)
- [Packages](https://buildkite.com/docs/packages)
- [APIs](https://buildkite.com/docs/apis)
- [Integrations](https://buildkite.com/docs/integrations)

Are you envisioning a new nav link?

Reply:
Probably need guidance from @gilesgas on that, but the content on this page atm doesn't really fit under pipelines

Reply:
Where are the docs for setting up organisations, teams, permissions etc?

Reply:
> The content on this page atm doesn't really fit under pipelines.

Yeah, you're right. It feels like we're missing a concept of a global/cross-product area

> Where are the docs for setting up organisations, teams, permissions etc?

At the moment those pages sit under the [Pipelines](https://buildkite.com/docs/pipelines) navigation link - https://buildkite.com/docs/team-management/permissions

Reply:
![CleanShot 2024-09-04 at 11 32 35](https://github.com/user-attachments/assets/21392531-fc49-4f72-bda7-1ecad1b4d46f)

Oh yeah, these are also definitely not pipelines haha

Reply:
Nope, they're not 😅😅